tests/StreamWrapper.php

```diff
@@ -90,20 +90,6 @@ public function stream_seek($offset, $whence)
      */
     public function stream_stat()
     {
-        return [
-            'dev'     => 0,
-            'ino'     => 0,
-            'mode'    => 33206,
-            'nlink'   => 0,
-            'uid'     => 0,
-            'gid'     => 0,
-            'rdev'    => 0,
-            'size'    => 0,
-            'atime'   => 0,
-            'mtime'   => 0,
-            'ctime'   => 0,
-            'blksize' => 0,
-            'blocks'  => 0,
-        ];
+        return fstat($this->stream);
```

Comment:
This is needed because the fake information was not right, so the stream_copy_to_stream didn't work. I could hardcode the 60 as size to pass the specific test but it could produce problems in future tests.